iphone/TitaniumKit/TitaniumKit/Sources/API/TiWindowProxy.m

```diff
@@ -558,6 +558,13 @@ - (void)openOnUIThread:(NSArray *)args
           [theController setModalPresentationStyle:style];
         }
       }
+
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+      if ([TiUtils isIOSVersionOrGreater:@"13.0"]) {
+        isForceModalDisable = ![TiUtils boolValue:@"forceModal" properties:dict def:NO];
```

Comment:
So it's creation only?

Reply:
This should be pass as parameter while opening window.